fusedrug_examples/interaction/drug_target/affinity_prediction/t5_mol/requirements.txt

```diff
@@ -0,0 +1,2 @@
+# new depdencies for T5 example
+sentencepiece
```

Comment:
Why do we have a requirement file here?
All the examples requirements are stored in a single file?

Reply:
Not needed - removed it